service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/dynamic/DynamicListElement.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.fpl.model.common.dynamic;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * An element of the {@link DynamicList}.
+ *
+ * <p>There are two properties which map to the relevant items of an option html tag.
+ */
+@Data
+@Builder
+public class DynamicListElement {
+
+    public static final String DEFAULT_CODE = "-1";
```

Comment:
👋

Reply:
As agreed this code is going to be removed.

Reply:
😢